### PR TITLE
Add BLL service unit test

### PR DESCRIPTION
**Pull request by blackberts**

### CODE REVIEW

File: BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    }
+
+    [Theory]
+    [InlineData("35edf9c3-7bb7-40b7-9ead-41ec4efec537")]
+    [InlineData("489e445b-e68b-4324-8b1e-4e2b01f03cdc")]
+    public async Task GetById_WhenEntityNotExist_ShouldReturnNull_Async(string id)
+    {
+        // Act
+        var guidId = Guid.Parse(id);
+        _bookingRepository.Setup(r => r.GetByIdAsync(guidId))
```
Comment (stas1408123) [inline BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs]: Its.Any<Guid>()

**Review by stas1408123 — CHANGES_REQUESTED** [review]

**Review by stas1408123 — APPROVED** [review]

File: BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs [hunk 2]
```diff
@@ -0,0 +1,198 @@
+﻿using FluentAssertions;
+
+namespace Booking.BLL.Tests.Services;
+
+public class BookingServiceTest
+{
+    private readonly Mock<IBookingRepository> _bookingRepository;
```
Comment (nantonov) [inline BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs]: Add postfix 'mock' to each mocked dependency variable name

File: BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+    public BookingServiceTest()
+    {
+        _bookingRepository = new Mock<IBookingRepository>();
+        _mapper = new Mock<IMapper>();
+        _bookingService = new BookingService(_bookingRepository.Object, _mapper.Object);
+    }
+
+    [Fact]
+    public async Task GetAll_WhenEntitiesExist_ShouldReturnListOfModels_Async()
```
Comment (nantonov) [inline BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs]: What _Async means?
Not needed I suppose

File: BKG/Booking.BLL/Abstractions/IGenericService.cs [hunk 4]
```diff
@@ -9,5 +9,5 @@ public interface IGenericService<TModel>
     Task<TModel> GetByIdAsync(Guid id);
     Task<TModel> AddAsync(TModel model);
     Task<TModel> UpdateAsync(TModel model);
-    Task DeleteAsync(Guid id);
+    Task<TModel> DeleteAsync(Guid id);
```
Comment (nantonov) [inline BKG/Booking.BLL/Abstractions/IGenericService.cs]: Get rid of Async postfix

**Review by nantonov — CHANGES_REQUESTED** [review]

File: BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs (hunk 3 above)
Comment (blackberts) [reply BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs]: All async methods need to have an Async postfix i think

File: .github/workflows/sonarcloud.yml [hunk 5]
```diff
@@ -40,8 +40,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          EXCLUDE_FROM_COVERAGE: " **/Controllers/**/*, **/Mapping/**/*, **/Validators/**/*, **/ViewModels/**/*, **/Controllers/**/*, **/Extensions/**/*, **/DI/**/*, **/Models/**/*, **/Abstractions/**/*, **/Configurations/**/*, **/Entities/**/*, **/Migrations/**/*"
```
Comment (nantonov) [inline .github/workflows/sonarcloud.yml]: Duplication of **/Controllers/**/*

File: BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs (hunk 3 above)
Comment (nantonov) [reply BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs]: No, not needed

**Review by nantonov — CHANGES_REQUESTED** [review]

**Review by nantonov — APPROVED** [review]

**Pull request merged**